src/Symfony/Component/Security/Core/Resources/translations/security.fa.xlf

```diff
@@ -52,19 +52,19 @@
             </trans-unit>
             <trans-unit id="13">
                 <source>Account has expired.</source>
-                <target>حساب کاربری منقضی شده است.</target>
+                <target>حساب کاربری منقضی گردیده است.</target>
             </trans-unit>
             <trans-unit id="14">
                 <source>Credentials have expired.</source>
-                <target>پارامترهای تعیین اعتبار منقضی شده‌اند.</target>
+                <target>مجوزهای احراز هویت منقضی گردیده‌اند.</target>
```

Comment:
Goole translates this as *"Authentication permissions have expired"*  I'm afraid that "credentials" is not the same as "permissions" in this case. Credentials = user + password, API token, certificate, etc. Permissions = ROLE_USER, etc.

Reply:
Hello @javiereguiluz .
Thank you for reviewing this request.

Reply:
Yes, I had doubts about the word 'Credentials' and did not change it in the previous one, but in this case its translation should be changed.
I am originally an Iranian, but I doubt about translating this.
In the previous case, the word 'Credentinal' is translated as 'Conditions' and is translated here as 'Parameters'.

Now, given that you said: Credentials = user + password, API token, certificate.
So I'm looking for a word that includes what you said?